speechbrain/lobes/models/huggingface_whisper.py

```diff
@@ -244,7 +245,7 @@ def _log_mel_spectrogram(self, audio):
         magnitudes = stft[..., :-1].abs() ** 2
 
         filters = self._mel_filters
-        mel_spec = filters @ magnitudes
+        mel_spec = filters.transpose(0, 1) @ magnitudes
```

Comment:
could you please elaborate on this?

Reply:
`filters.T @ magnitudes` also implies the same.

Reply:
Yes. But if you could check what changed in Transformer it would be really helpful. Furthermore, have you tried to run the Whisper in CommonVoice or LibriSpeech for instance? Do you get the expected WER?

Reply:
> have you tried to run the Whisper in CommonVoice or LibriSpeech for instance? Do you get the expected WER?
I have only tried debugging. Running the entire expt would be quite a time and resource-consuming.  I can try it later on, since I am bit busy with the RescueSpeech recipe.